easy-ui-react/src/Select/Select.tsx

```diff
@@ -0,0 +1,166 @@
+import React, { Key, useMemo } from "react";
+import { CollectionChildren } from "@react-types/shared";
+import { useSelectState } from "react-stately";
+import { AriaSelectProps, useSelect } from "react-aria";
+import { InternalSelectContext } from "./SelectContext";
+import { SelectField, SelectFieldProps } from "./SelectField";
+import { SelectOption } from "./SelectOption";
+import { SelectSection } from "./SelectSection";
+import { SelectOverlay } from "./SelectOverlay";
+import { useTriggerWidth } from "../Menu/useTriggerWidth";
+
+export type SelectProps<T> = AriaSelectProps<T> &
+  SelectFieldProps & {
+    /** Method that is called when the open state of the select field changes. */
+    onOpenChange?: (isOpen: boolean) => void;
+    /** Sets the open state of the select field. */
+    isOpen?: boolean;
```

Comment:
not sure if it makes sense for a `Select` to have these handlers; i could go either way on this